airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -78,6 +78,13 @@ class KubernetesPodOperator(BaseOperator):
         /airflow/xcom/return.json in the container will also be pushed to an
         XCom when the container completes.
     :type xcom_push: bool
+    :param is_delete_operator_pod: Delete the pod after it completes
+    :type is_delete_operator_pod: bool
+    :param keep_failed_pod: Keep pods that exit with non zero error code. This allows
+                            for easy log inspection of failed pods - assuming the
+                            node is still in the cluster
+                            (only works when is_delete_operator_pod is True)
+    :type keep_failed_pod: bool
     :param tolerations: Kubernetes tolerations
     :type list of tolerations
```

Comment:
Actually this line `:type list of tolerations` was incorrect. Would be good if you can fix it alongside.